src/DayColumn.js

```diff
@@ -160,31 +160,31 @@ class DaySlot extends React.Component {
       availabilityWrapperComponent: AvailabilityWrapper,
       availabilityStartAccessor,
       availabilityEndAccessor,
+      entityKeyAccessor,
       min,
-      date,
     } = this.props;
     let AvailabilityComponent = availabilityComponent;
     let styledAvailabilities = getStyledAvailabilities({
-      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin, date
+      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin
     });
 
-    return styledAvailabilities.map(({availability, style}, index) => {
+    return styledAvailabilities.map(({availability, style}, idx) => {
       let { height, top } = style;
+      const key = entityKeyAccessor && availability[entityKeyAccessor]
+        ? availability[entityKeyAccessor]
```

Comment:
This doesn't work because `entityKeyAccessor` is for telling the calendar how to grab the key for an event, not an availability. We'd have to use a new prop like `availabilityKeyAccessor`.